cloud-data-protection/src/components/navMenu.tsx

```diff
@@ -0,0 +1,82 @@
+import React, {Fragment} from 'react';
+import {useSelector} from "react-redux";
+import {selectAuthenticated, selectUser} from "../features/userSlice";
+import {
+    Home,
+    AccountCircle,
+    Person,
+    PersonAdd,
+    ExitToApp
+} from '@material-ui/icons';
+import {
+    AppBar, Divider,
+    Drawer,
+    ListItem,
+    ListItemIcon,
+    ListItemText,
+    Toolbar,
+} from "@material-ui/core";
+import './navMenu.css';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/olivierbouchomsfontys/cloud-data-protection/pullRequest?prid=7219402)
```suggestion
import "./navMenu.css";
```